bazel/test.bzl

```diff
@@ -341,7 +340,7 @@ def redpanda_test_cc_library(
         defines = defines,
         local_defines = local_defines,
         visibility = visibility,
-        include_prefix = include_prefix,
+        include_prefix = native.package_name().removeprefix("src/v/"),
```

Comment:
neat trick

Reply:
One Neat Trick to reduce your BUILD file boilerplate...